custom-metrics-api/custom-metrics-config-map.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: adapter-config
+  namespace: monitoring
+data:
+  config.yaml: |
+    rules:
+    - seriesQuery: '{__name__=~"^container_.*",container_name!="POD",namespace!="",pod_name!=""}'
```

Comment:
What does this query? Maybe add a comment?

Reply:
This config was earlier bundled, they now decided to extract that as a config if you'd like to modify. We are still using the default